src/plugins/irc-events/connection.js

```diff
@@ -48,6 +48,11 @@ module.exports = function(irc, network) {
 	});
 
 	irc.on("socket connected", function() {
+		network.prefixLookup = {};
+		_.each(irc.network.options.PREFIX, function(mode) {
```

Comment:
Might be a nitpick, but any reason to use Lodash's `.each` instead of native `forEach`?